### DISCUSSION TITLE
Unexpected exception caught, the kernel will be rebooted.

### DISCUSSION THREAD

**Discussion by QuietImprovement** [roadrunner-server/roadrunner#1837] [Q&A] [answered]
Greetings, I've noticed in my logs that Roadrunner has encountered the following situation multiple times: "Unexpected exception caught, the kernel will be rebooted" Is there a prescribed method for throwing exceptions with Roadrunner to prevent a kernel reboot?

**Comment by rustatian** [comment]
Hey @achrefriahi-limango 👋 
I guess this question should be addressed to the developers of the framework you use. Do you use Symfony or Laravel or..?

**Reply by QuietImprovement** [reply, marked as answer]
Hi, @rustatian,
Symfony, I found the reason, i will share some links, maybe it can help someone else
[https://github.com/Baldinof/roadrunner-bundle?tab=readme-ov-file#kernel-reboots](https://github.com/Baldinof/roadrunner-bundle?tab=readme-ov-file#kernel-reboots)
[https://github.com/Baldinof/roadrunner-bundle/issues/24](https://github.com/Baldinof/roadrunner-bundle/issues/24)

**Reply by rustatian** [reply]
Cool, I'll add these links to the docs 👍
